prov/usnic/src/usdf_cq.c

```diff
@@ -92,8 +92,9 @@ usdf_cq_readerr(struct fid_cq *fcq, struct fi_cq_err_entry *entry,
 	cq = container_of(fcq, struct usdf_cq, cq_fid);
 
 	// If top entry has no error, return 0
```

Comment:
This comment no longer agrees with the (otherwise good) change you made.  Please update/strike the comment.